### PR TITLE
add configurable picture path in preprare for work

### PR DESCRIPTION
**Pull request by lucienicolier**

### CODE REVIEW

File: en/prepare/picture_path.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+Configurable picture path
+=========================
+
+QFieldSync gives the possibility to configure the path of the attachments.
```
Comment (m-kuhn) [inline en/prepare/picture_path.rst:5]:
```suggestion
QFieldSync gives the possibility to configure the path of picture attachments.
```

Let's be consistent in here (picture, photo, attachments - all refers to the same)

File: en/prepare/picture_path.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+1. Go to QFieldSync plugin --> Project configuration
+2. Select the table "Photo Naming"
+3. Choose the layer, the field and introduce the expression
```
Comment (m-kuhn) [inline en/prepare/picture_path.rst:9]:
```suggestion
3. Choose the layer, the field and configure the expression 
```

**Review by m-kuhn — COMMENTED** [review]
Thanks

I think "synchronize the files ..." and "fix adding project folder" can be removed. They were important fixes (and therefore mentioned in the changelog) but nothing to add to the documentation.

File: en/prepare/picture_path.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+Configurable picture path
+=========================
+
+QFieldSync gives the possibility to configure the path of the attachments.
+
+1. Go to QFieldSync plugin --> Project configuration
+2. Select the table "Photo Naming"
+3. Choose the layer, the field and introduce the expression 
+
+Use expressions to configure the path of the attachments. Usually DCIM is used as the main folder and it's the default.
```
Comment (m-kuhn) [inline en/prepare/picture_path.rst:11]:
```suggestion
Use expressions to configure the path of the attachments. By default, pictures are saved into the `DCIM` folder with a timestamp as name.
```

**Comment by m-kuhn** [comment]
> I think "synchronize the files ..." and "fix adding project folder" can be removed.

Can you take care of that @lucienicolier ?

**Comment by lucienicolier** [comment]
@m-kuhn it's done

**Review by m-kuhn — APPROVED** [review]
Thank you @lucienicolier 
please merge

**Pull request merged**